app/routes/index.tsx

```diff
@@ -129,9 +130,15 @@ export default function App() {
       return
     }
 
+    const url = new URL(el.href)
     const href = el.href.replace(/\?.*$/, '')
     const found = onSiteGDocLinkMapRef.current[href]
-    if (found) {
+
+    if (url.hostname === window.location.hostname) {
+      e.preventDefault()
+      const state = new URLSearchParams(url.search).get('state')
+      if (state) selectQuestion(getStateEntries(state)[0][0], '')
+    } else if (found) {
```

Comment:
ah, so we will support both for now, then migrate the parser, then remove the old logic, nice 👍